src/components/ThemeToggle/ThemeToggle.test.tsx

```diff
@@ -80,23 +83,63 @@ describe('calling hook functions', () => {
 
   test('calls setTheme when the toggle is clicked', async () => {
     const user = userEvent.setup()
+
+    renderWithAuthAndApollo(
+      <ThemeProvider enableSystem={false}>
+        <ThemeToggle />
+      </ThemeProvider>,
+      {},
+      editThemeMock
+    )
+
     const toggle = screen.getByTestId('theme-toggle')
     await user.click(toggle)
     expect(setItemMock).toHaveBeenCalledTimes(1)
     expect(setItemMock).toHaveBeenCalledWith('theme', 'dark')
-
-    await user.click(toggle)
-    expect(setItemMock).toHaveBeenCalledTimes(2)
-    expect(setItemMock).toHaveBeenLastCalledWith('theme', 'light')
   })
 
   test('calls handleEditThemeMutation when the toggle is clicked', async () => {
     const user = userEvent.setup()
+
+    renderWithAuthAndApollo(
+      <ThemeProvider enableSystem={false}>
+        <ThemeToggle />
+      </ThemeProvider>,
+      {},
+      editThemeMock
+    )
+
     const toggle = screen.getByTestId('theme-toggle')
     await user.click(toggle)
 
     // only need to check that the function was called the expected number of times
     // the expected parameters are defined in the mock
     expect(editThemeMock[0].newData).toHaveBeenCalledTimes(3)
   })
+
+  test('throws an error', async () => {
+    const consoleSpy = jest.spyOn(console, 'error')
+
+    const errorMock = [
+      {
+        request: {
+          query: GetThemeDocument,
+        },
+        error: new Error('Test error'),
+      },
+    ]
+
+    renderWithAuth(
+      <MockedProvider mocks={errorMock} addTypename={false}>
+        <ThemeToggle />
+      </MockedProvider>
+    )
+
+    waitFor(() =>
+      expect(consoleSpy).toHaveBeenCalledWith(
+        'Error updating theme: error in handleThemeChangeAndTracking',
+        errorMock[0].error
+      )
+    )
+  })
```

Comment:
I added this because in the test coverage report, the only line not being tested in `ThemeToggle` was the error state. For some reason, this did not bump up the test coverage, but I decided to leave it in anyways.